public/js/components/Menu.jsx

```diff
@@ -46,6 +47,9 @@ var Menu = React.createClass({
                       unleash admin
                    </span>
                   </a>
+                  <div style={{position: "absolute", right: "0"}}>
```

Comment:
Looks like there are no good way to do this in Spaden. What about adding this style to the [custom css](https://github.com/finn-no/unleash/blob/master/public/css/unleash.css) and adding a class.